app/server.go

```diff
@@ -0,0 +1,21 @@
+package app
+
+import (
```

Comment:
As a convention, the imports should be separated as follows:
```go
import (
    Go core
    
    Local
    
    ThirdParty
)
```

Do this in all the imports.